sdk-java/src/main/java/io/littlehorse/sdk/wfsdk/internal/WorkflowThreadImpl.java

```diff
@@ -444,19 +452,42 @@ public SpawnedThreadImpl spawnThread(ThreadFunc threadFunc, String threadName, M
         return new SpawnedThreadImpl(this, threadName, internalStartedThreadVar);
     }
 
+    public void overrideTaskRetries(TaskNodeOutputImpl node, int retries) {
+        checkIfIsActive();
+        Node.Builder nb = spec.getNodesOrThrow(node.nodeName).toBuilder();
+        if (nb.getNodeCase() != NodeCase.TASK) {
+            throw new IllegalStateException("Impossible to not have task node here");
```

Comment:
hey @coltmcnealy-lh what do you think about having a kind of positive statement for this error message, something similar to : "only tasks nodes allowed", I think is easier from the user perspective. wdyt?

Reply:
It's not possible to construct an input that makes this not be a task node. Which means that if we reach here, then there is a bug in the SDK itself.